adapter/pkg/logging/logging_constant.go

```diff
@@ -0,0 +1,266 @@
+/*
+ *  Copyright (c) 2021, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package logging
+
+// Error Log Pkg operator(2600-2699) Config Constants
+const (
```

Comment:
do we require these constants? Is their any usefulness of it? Shall we remove these?

Reply:
This is for the developers to identify which codes are implemented already.